dockerfile/moneo-exporter-nvidia.dockerfile

```diff
@@ -1,29 +1,45 @@
-FROM nvidia/cuda:11.1.1-runtime-ubuntu18.04
+FROM nvcr.io/nvidia/cuda:12.2.2-runtime-ubuntu22.04
 
 LABEL maintainer="Moneo"
 
 ARG BRANCH_OR_TAG=main
+ARG DCGM_VERSION=3.1.1
 ENV PROFILING false
+ENV GPU_SAMPLE_RATE 2
+
+ENV DEBIAN_FRONTEND=noninteractive
```

Comment:
Was there a package that was prompting a window/gui to pop up?

Reply:
It's for python3.10 installation when using 1804 as a base image, it would be useless after changing the base image to 2204, so it is a good idea to remove this line. Thanks!